Plugins/Gradle/src/main/kotlin/dev/testify/TestifyPlugin.kt

```diff
@@ -127,6 +129,7 @@ class TestifyPlugin : Plugin<Project> {
     }
 
     companion object {
+        var consoleColorEnabled = true
```

Comment:
@DanielJette I don't like this at all, but didn't like passing around project either. Have any suggestions or alternatives? `TestifySettings`?

Reply:
I looked into this shortly after you originally suggested using the console mode and came to the same conclusion. With the architecture of the gradle plugin, I don't think there's another choice.

This is consistent with other solutions we're using in the plugin, at least.